recipes/gimmemotifs/meta.yaml

```diff
@@ -42,7 +42,7 @@ requirements:
     - pyarrow
     - pybedtools
     - pysam
-    - python >=3.6
+    - python
```

Comment:
`conda-build` automatically ties the `python` version in `requirements/run` to that in `requirements/host`, i.e., this will become `python >=3.6,<3.7.0a0`.
Allowing `python >=3.6`, e.g., `python 3.7` and onward, does not make sense since the `lib/pythonX.Y/` paths that the package gets build for are version-dependent.